tests/testsuite/alt_registry.rs

```diff
@@ -706,11 +707,6 @@ fn no_api() {
     // Check all of the API commands.
     let err = "[ERROR] registry `alternative` does not support API commands";
 
-    p.cargo("login --registry alternative TOKEN")
-        .with_status(101)
-        .with_stderr_contains(&err)
-        .run();
```

Comment:
I should be able to put it back. Thanks for asking.

`cargo login` fetches `config.json` from the registry index in order to determine the API url so it can construct the `/me` URL. That failure could have been caused by missing token. Now that we have the `AuthorizationError` error type, I think I can specifically detect that case, and allow this to become a failure again.